awscli/topics/config-vars.rst

```diff
@@ -200,8 +200,18 @@ You can specify the following configuration values for configuring an IAM role
 in the AWS CLI config file:
 
 * ``role_arn`` - The ARN of the role you want to assume.
-* ``source_profile`` - The AWS CLI profile that contains credentials we should
-  use for the initial ``assume-role`` call.
+* ``source_profile`` - The AWS CLI profile that contains credentials /
+  configuration we should use for the initial ``assume-role`` call. This
```

Comment:
Probably want to remove the use of "we" to make it more third person or instructive.